cmd/notation/policy/import.go

```diff
@@ -0,0 +1,110 @@
+package policy
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation-go/verifier/trustpolicy"
+	"github.com/notaryproject/notation/internal/cmd"
+	"github.com/spf13/cobra"
+)
+
+type importOpts struct {
+	cmd.LoggingFlagOpts
+	filePath string
+	force    bool
+}
+
+func importCmd() *cobra.Command {
+	var opts importOpts
+	command := &cobra.Command{
+		Use:   "import [flags] <file_path>",
+		Short: "[Preview] Import trust policy configuration from a JSON file",
+		Long: `[Preview] Import trust policy configuration from a JSON file.
+
+** This command is in preview and under development. **
+
+Example - Import trust policy configuration from a file:
+  notation policy import my_policy.json
+`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.filePath = args[0]
+			return runImport(cmd, opts)
+		},
+	}
+	command.Flags().BoolVar(&opts.force, "force", false, "override the existing trust policy configuration, never prompt")
+	opts.LoggingFlagOpts.ApplyFlags(command.Flags())
+	return command
+}
+
+func runImport(command *cobra.Command, opts importOpts) error {
+	// optional confirmation
+	var mode os.FileMode = 0644
+	policyPath := filepath.Join(dir.UserConfigDir, dir.PathTrustPolicy)
+	if fi, err := os.Stat(policyPath); err == nil {
+		mode = fi.Mode()
```

Comment:
`LoadDocument` seems too heavy for detecting whether a file exists or not, also the permission information is not viable on caller side.

Reply:
It would be great if notation-go have a `LoadDocumentInPath` which can be used to read in the to-be-imported policy file

Reply:
Will make imported config permission to 600.

But still it's wrong to use `LoadDocument()` here on the to-be-replaced config. Consider this scenario: if user have a malformed trust policy file `A` in the notation config path and want to import a valid policy `B` in. Use `LoadDocument()` here will 1) open `A`, (which is unnecessary since checking the existence of a file only requires open its containing folder); and 2) always fail because `A` is malformed. So the result is **user will not be able to import a valid policy since current policy is malformed**.

Reply:
> Will make imported config permission to 600.
> 
> But still it's wrong to use `LoadDocument()` here on the to-be-replaced config. Consider this scenario: if user have a malformed trust policy file `A` in the notation config path and want to import a valid policy `B` in. Use `LoadDocument()` here will 1) open `A`, (which is unnecessary since checking the existence of a file only requires opennig its containing folder); 

As I mentioned above, we are not just checking the existence here. If the original trust policy cannot even pass `LoadDocument()` then we should allow user to override it directly. Because otherwise, the user might think there's already a working trust policy doc in the configuration and stop the import at confirmation.

> and 2) always fail because `A` is malformed. So the result is **user will not be able to import a valid policy since current policy is malformed**.

my code suggestion was: `if _, err := LoadDocument(); err == nil {`, if the original doc can pass `LoadDocument()`, then we need to prompt to ask for confirmation. If `err != nil`, we just go ahead and import the new policy.

Reply:
> If the original trust policy cannot even pass LoadDocument() then we should allow user to override it directly

Sounds workable. Will change accordingly. Also the scope of `--force` need to be expanded and if it's set, policy loading will be bypassed.